backend/src/main/java/de/sambam/gamenighttracker/controller/UserController.java

```diff
@@ -24,10 +21,24 @@ public UserController(UserService userService) {
         this.userService = userService;
     }
 
-    @GetMapping({"/games"})
-    public List<Game> gameList() {
-        return userService.listAllGames("1")
-                .orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "User not found"));
+    @GetMapping("/games")
+    public List<Game> getGameList() {
+        return userService.listAllGames("1");
```

Comment:
You want to later get the player id from the principal, I guess?

Reply:
probably ^^'